src/app/BUILD.gn

```diff
@@ -103,14 +103,6 @@ static_library("app") {
     "reporting/Engine.h",
   ]
 
-  if (chip_ip_commissioning) {
-    defines = [
-      "CONFIG_USE_CLUSTERS_FOR_IP_COMMISSIONING=1",
-      "CHIP_DEVICE_CONFIG_ENABLE_EXTENDED_DISCOVERY=1",
-      "CHIP_DEVICE_CONFIG_ENABLE_JUST_IN_TIME_PROVISIONING=1",
```

Comment:
Is the intent that people should be defining those two options separately now instead of having IP commissioning imply them?

Reply:
These were here as stop-gaps because of the issue with how we were determining whether a device was commissioned (before we had op-certs working). It forces the device to continue advertising as commissionable even if it thinks it's already been commissioned. Previously, that was determined by whether we had network access, which we clearly did for on-network provisioning. Now that's fixed, so this stop-gap is no longer required. We will advertise as commissionable if we don't have certs, even if we have network access. Manufacturers can decide for themselves whether to enable extended discovery.